Codereview branch

asadmisrani opened the pull request:
All files are added

asadmisrani committed:
All files are added

asadmisrani committed:
Style.css file was modified

asadmisrani commented on README.md:
```diff
@@ -1 +1,251 @@
-# empty-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```
Please remove all the commented-out codes from the readme file before submitting them to your repository. 😄 because commented code makes your codebase populated and it is against clean code

asadmisrani commented on README.md:
```diff
@@ -1 +1,251 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MY PORTFOLIO] <a name="about-project"></a>
+
+My name is Esmael Mohammed, this project is my portfolio webpage project
+
+**[MY PORTFOLIO]** A responsive website called MY PORTFOLIO accurately describes who I am, my professional accomplishments, and my job history.
+
```
You have not used your real name in the My Portfolio section in the README file. kindly consider using your real name. it will make your repo professional

asadmisrani commented on README.md:
```diff
@@ -1 +1,251 @@
+## 👥 Authors <a name="authors"></a>
+
+ 
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/esmiz-dream)
+- Linkedin: [@githubhandle](linkedin.com/in/esmael-mohammed-10ba5a197)
+ 
```
Please use your correct information in the author section, link your LinkedIn profile, and link your GitHub profile 👤. It will make your repo professional and the visitor can contact you through these links.

asadmisrani commented on styles.css:
```diff
@@ -0,0 +1,1110 @@
+  padding-left: 0;
+}
+
+/* main content starts here */
+.main-content {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  width: 375px;
+  flex-direction: column;
+  position: relative;
+  background: #fff;
+}
+/* main content ends here  */
```
Please do not leave unused code as a commit in your codebase. Unused code as commits makes your code not clear. 😸

asadmisrani commented on styles.css:
```diff
@@ -0,0 +1,1110 @@
+  color: #fff;
+}
+
+.common-work-card button {
+  position: relative;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+  border: none;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+/* list styling for my-work section */
+.my-work-grid-container ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  column-gap: 12px;
+}
+
```
Please create your button dynamically by using two for-loops. ⏺️

Zuhaib042 replied:
- [ ] Kindly consider explaining why are you asking for these changes so that the student can understand in a better way.

asadmisrani reviewed (commented):
# Changes required ♻️ 

-Please check out my commits below and make all required changes.

-Submit for a re-review when you are done.

-Thanks 👍

joel-kalema commented on index.html:
```diff
@@ -0,0 +1,318 @@
+<!DOCTYPE html>
+<html lang="en">
```
- [ ] Good job using inline comments 👍 , but you need to fix some parts 😃   
- [ ]  Kindly , make sure to highlight the effort of the student 👍
- [ ]  I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.

Please consider always including an explanation, a "why", for each requested change. This means to describe to the user why he is being asked for some change.

#### General example:-

  - **This is a well-explained request:** Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
  - **This request lacks a "why":** Why don't you use an article tag here instead of a div?

asadmisrani replied:
I have already fixed some parts which you mentioned in the commit.

I highlighted to students in commits.

I have added emojis to my comments for the student so that he will understand more.

Thanks 👍

joel-kalema reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @asadmisrani 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
Your comments are clear and easy to understand 💯 ✔️

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani reviewed (commented):
To highlight 🍾
All linters are passing ✔️
Your comments are clear and easy to understand 💯 ✔️

-Check the comments under the review. 👍 

Thanks 👍

asadmisrani commented:
I have already fixed some parts which you mentioned in the commit.

I highlighted to students in commits.

I have added emojis to my comments for the student so that he will
understand more.

Thanks 👍

On Tue, Mar 28, 2023 at 9:49 PM joel kalema ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRED CHANGES ♻️♻️
>
> Hi @asadmisrani <https://github.com/asadmisrani>
> Good job so far! I know you worked hard to implement this project, I
> really appreciate the effort you put in! But there are some issues that you
> still need to work on to go to the next project but you are almost there!
>
> [image: almost-there]
> <https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif>
> To highlight 🍾
>
>    - All linters are passing ✔️
>    Your comments are clear and easy to understand 💯 ✔️
>
> Required changes ♻️ ♻️
>
>    - Check the comments under the review.
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
>
> *Feel free to leave any questions or comments in the PR thread if
> something is not 100% clear. Please ping me @joel-kalema
> <https://github.com/joel-kalema> when you comment so I can receive the
> notification or use slack with the same name.*
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/empty-/pull/1#discussion_r1150900376>:
>
> > +<!DOCTYPE html>
> +<html lang="en">
>
>
>    - Good job using inline comments 👍 , but you need to fix some parts 😃
>    - Kindly , make sure to highlight the effort of the student 👍
>    - I suggest you add emoji's to your comments so that the student
>    understands the tone you are trying to communicate 😎.
>
> Please consider always including an explanation, a "why", for each
> requested change. This means to describe to the user why he is being asked
> for some change.
> General example:-
>
>    - *This is a well-explained request:* Why don't you use an article tag
>    for the .first-article element instead of a div, that would improve the
>    HTML structure and make the page more accessible
>    - *This request lacks a "why":* Why don't you use an article tag here
>    instead of a div?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/empty-/pull/1#pullrequestreview-1361528533>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKEFRROCGGRBFTZ7LHDW6MJBZANCNFSM6AAAAAAWKY6ZCU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Zuhaib042 commented on README.md:
```diff
@@ -1 +1,251 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```
- [ ] Kindly consider explaining why the suggested changes are required as suggested by the previous reviewer.
you can write `Please remove all the commented-out code from the readme file before submitting them to your repository. 😄 because commented code make your codebase populated and it is against clean code`

asadmisrani replied:
Thanks for letting me know, I already edited the message to the student. 👍

Zuhaib042 commented on README.md:
```diff
@@ -1 +1,251 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MY PORTFOLIO] <a name="about-project"></a>
+
+My name is Esmael Mohammed, this project is my portfolio webpage project
+
+**[MY PORTFOLIO]** A responsive website called MY PORTFOLIO accurately describes who I am, my professional accomplishments, and my job history.
+
```
- [ ] Here also kindly consider explaining the why. you can write `You have not used your real name in the My Portfolio section in the README file. kindly consider using your real name. it will make your repo professional`

asadmisrani replied:
Thanks for letting me know, I already edited the message to the student. 👍

Zuhaib042 commented on README.md:
```diff
@@ -1 +1,251 @@
+
+ 
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/esmiz-dream)
+- Linkedin: [@githubhandle](linkedin.com/in/esmael-mohammed-10ba5a197)
+ 
```
- [ ]  Here also kindly consider explaining the why. you can write `Please use your correct information in the author section, link your LinkedIn profile, and link your GitHub profile 👤. It will make your repo professional and the visitor can contact you through these links.`

asadmisrani replied:
Thanks for letting me know, I already edited the message to the student. 👍

Zuhaib042 reviewed (changes requested):
Hi @asadmisrani , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![](https://media.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif)



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani reviewed (commented):
# Changes required ♻️ 

Please kindly check my comments below some changes are required, because it's very important to you to know how to write proper code in your codebase, it's required in your future projects. 📽️ 

Thanks 👍

Shisui6 reviewed (approved):
Hi @asadmisrani, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good summary ✅
- Good use of inline comments ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani commented:
Approved.
